src/ops/info.rs

```diff
@@ -154,12 +154,12 @@ fn find_pkgid_in_summaries(
             let s1_matches = s1
                 .as_summary()
                 .rust_version()
-                .map(|v| v.to_caret_req().matches(rustc_version))
+                .map(|v| v.as_partial().to_caret_req().matches(rustc_version))
```

Comment:
I'd need to check when i have access to docs but i suspect this was after i added an api for msrv checks and we should use that instead

Reply:
Yes. I will update it.